contracts/tokens/Cruzo1155.sol

```diff
@@ -28,7 +30,7 @@ contract Cruzo1155 is ERC1155URI {
      *
      */
     function total() public view onlyOwner returns (uint256) {
```

Comment:
We can remove this view, it's not used

Reply:
we can turn it to external view, but some tests requires this func